catalyst_voices/lib/widgets/buttons/specialized/voices_learn_more_button.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:catalyst_voices/widgets/widgets.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:flutter/material.dart';
+
+/// A "Learn More" button that redirects usually to an external content.
+class VoicesLearnMoreButton extends StatelessWidget {
```

Comment:
Is `voices_buttons.dart` better place for such button?

eg. https://github.com/input-output-hk/catalyst-voices/pull/881/files#diff-a9d554894c1a22e2e2e04448cab4a67fd40a1c6fbd490569cac61cb009f28784R152

Reply:
Agree, moved there